azuredevops/internal/service/approvalsandchecks/resource_check_required_template.go

```diff
@@ -0,0 +1,104 @@
+package approvalsandchecks
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/microsoft/terraform-provider-azuredevops/azuredevops/utils/pipelineschecksextras"
+)
+
+var validRepositoryTypes = []string{"git", "github", "bitbucket"}
```

Comment:
Can we rename `git` to `azuregit`? `git` and `github` will confuse customers.

Reply:
done